frontend/src/app/components/ExperimentPage.js

```diff
@@ -90,6 +90,14 @@ export default class ExperimentPage extends React.Component {
     return currentExperiment[0];
   }
 
+  isValidVersion(min) {
+    const version = parseInt(navigator.userAgent.split('/').pop(), 10);
+    if (typeof min === 'undefined' || version >= min) {
```

Comment:
We can just `return typeof min === 'undefined' || version >= min;` here, no reason for the conditional.